news/_posts/core-weekly/2019-03-13-coreweekly-week-10-2019.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 10 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-03-13 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 04th to Sunday 10th of March 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+An important step for the PrestaShop open source project has been completed last week and it is available today: the nightly builds and tests dashboard. This is a new website: https://nightly.prestashop.com/.
```

Comment:
It's better for SEO to put links together with text:
```suggestion
An important step for the PrestaShop open source project has been completed last week and it is available today: the [PrestaShop nightly builds and tests dashboard](https://nightly.prestashop.com/).
```

Reply:
I know but this is a new URL and I would like to introduce it.